### PR TITLE
A workshop exercise of how to use useQueryLoader and usePreloadedQuery

### PR DESCRIPTION
**Pull request by arantespp**
# Summary

An example of how to use `userQueryLoader` + `usePreloadedQuery` with a search case.

## Test Plan

- `cd workshop/13-useQueryLoader`
- `yarn run copy-env`
- `yarn`
- `yarn start`

### What's required for testing (prerequisites)?

This will only work if the tasks were done.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -118,5 +118,9 @@
       "workshop-solutions/*",
       "slides/*"
     ]
+  },
+  "dependencies": {
+    "react": "^0.0.0-experimental-4ead6b530",
+    "react-dom": "^0.0.0-experimental-4ead6b530"
```
Comment (sibelius) [inline package.json:124]: why we need this?

File: packages/server/schema/schema.graphql [hunk 2]
```diff
@@ -118,41 +179,63 @@ type CommentConnection {
...
+  mongoose _id
+  """
   _id: String!
   body: String
   user: User
   post: Post
   likesCount: Int!
 
-  """whether logged user liked this post"""
+  """
```
Comment (sibelius) [inline packages/server/schema/schema.graphql:236]: why schema changed?

Comment (arantespp) [reply packages/server/schema/schema.graphql:236]: My VSCode automatic Eslint made these changes.

Comment (arantespp) [reply packages/server/schema/schema.graphql:236]: Also, I've created a query `searchPosts` to relay-compile get the types.

File: package.json (hunk 1 above)
Comment (arantespp) [reply package.json:124]: Because I was getting an error that I only know how to fix running `yarn add -W react@experimental react-dom@experimental`

Comment (sibelius) [reply package.json:124]: what was the error?

this could break the workshop

File: packages/server/schema/schema.graphql (hunk 2 above)
Comment (sibelius) [reply packages/server/schema/schema.graphql:236]: you can regenerate using `yarn update-schema`

File: package.json (hunk 1 above)
Comment (arantespp) [reply package.json:124]: I see. Is it better to remove concurrent mode (`useTransition`) or just move these installations to the local package "@workshop/13-useQueryLoader"?

Comment (sibelius) [reply package.json:124]: what is the errors?

File: packages/server/schema/schema.graphql (hunk 2 above)
Comment (arantespp) [reply packages/server/schema/schema.graphql]: Done! I forgot you create the schema via code. I use to change the `graphql.schema` directly.

File: package.json (hunk 1 above)
Comment (arantespp) [reply package.json]: I remove the dependencies from root `package.json`, run `yarn --force` and it worked.

**Pull request closed without merging**